### PR TITLE
Carousel container

### LINKED ISSUE #9: As user I want to see recipe cards in cinema slide container

### PR DESCRIPTION
**Pull request by ReebalSami**

### CODE REVIEW

File: frontend/src/components/RecipeCard/RecipeCard.tsx [hunk 1]
```diff
@@ -14,9 +19,33 @@ export default function RecipeCard(props: Readonly<RecipeCardProps>) {
...
-            <p>{props.recipe.description}</p>
-        </button>
-    )
+        <Card className="recipe-card">
+            <CardActionArea>
+                <CardMedia
+                    component="img"
+                    height="140"
+                    image= {props.recipe.imageUrl}
+                    alt= "https://res.cloudinary.com/dergdq6qs/image/upload/v1712181382/o5gm3luzvcnv0ifsznnw.jpg"
```
Comment (MarcellNF) [inline frontend/src/components/RecipeCard/RecipeCard.tsx]: what is this alt text? you could say as alt text the recipe name for example, but not linking an image

**Review by MarcellNF — CHANGES_REQUESTED** [review]
One little thing :-)

File: frontend/src/components/RecipeCard/RecipeCard.tsx (hunk 1 above)
Comment (ReebalSami) [reply frontend/src/components/RecipeCard/RecipeCard.tsx]: I want to make an alternative image. I didn't know for now how to do it, so I kept the Link of the alternative image in the alt, not to lose it :)

Comment (MarcellNF) [reply frontend/src/components/RecipeCard/RecipeCard.tsx]: if you want an alternative image you can use the ternary operator: `props.recipe.imageUrl ? your-card-media-with-image : your-alternative-card-media-with-alternative-image`

This is not a good practice and if you want to save it for later, you should have a document where you can save things like this

Comment (MarcellNF) [reply frontend/src/components/RecipeCard/RecipeCard.tsx]: the alt-attribute does not replace your image. its an alternative if the image can not be loaded for some reason. than there will be a text

Comment (ReebalSami) [reply frontend/src/components/RecipeCard/RecipeCard.tsx]: It worked! thx for the hint!!

**Review by MarcellNF — APPROVED** [review]
LGTM!:)

**Pull request merged**